cardano-node/app/cardano-cli.hs

```diff
@@ -1,208 +1,39 @@
 {-# LANGUAGE NamedFieldPuns #-}
+module Main (main) where
 
 import           Cardano.Prelude hiding (option)
-import           Prelude (String, error, id)
+import           Prelude (String)
 
-import           Cardano.Binary (Annotated (..))
-import           Cardano.Chain.Common
-import qualified Cardano.Chain.Genesis as Genesis
-import           Cardano.Chain.Slotting (EpochNumber(..))
-import           Cardano.CLI.Run (ClientCommand(..), SystemVersion(..), decideKeyMaterialOps, runCommand)
-import           Cardano.Crypto (AProtocolMagic(..), ProtocolMagic, ProtocolMagicId(..), RequiresNetworkMagic(..))
-import           Cardano.Node.CLI (command')
+import           Cardano.CLI.Ops (decideCLIOps)
+import           Cardano.CLI.Run (ClientCommand(..), runCommand, parseClientCommand)
```

Comment:
`parseClientCommand` belongs with the other parsers used only in `cardano-cli.hs`. It's less overhead when the parsers are just below where they are called. If the parser is used in multiple places then it can go into `Cardano.Node.Parsers`.

Reply:
The thing is, it's also less overhead to have the only three places that mention `ClientCommand` in one place. The `main` in `cardano-cli.hs` only just refers to it -- where as `CLI.Run` has both the definition of `ClientCommand` and `runCommand` that interprets it.  Adding in the parser simply completes the triumvirate.

I.e. to add/modify one branch, you can stay entirely within a single file.

What do you think, @Jimbo4350?

Reply:
I understand your point but for now I think the style is clearer and more consistent if _all_ the exec's have their parsers in their respective 'main' file.

Reply:
@Jimbo4350, done!